mints/06.json

```diff
@@ -0,0 +1 @@
+{"name":"contract/valory/fpmm:0.1.0","description":"The scaffold contract scaffolds a contract to be implemented by the developer.","code_uri":"ipfs://bafybeihmzrpamqh654kb2vzzybzkirk75rmxj6rttrbhhkrju73jyjrjse","image":"ipfs://bafybeiabcpurda2ownkndcpchn5rtlwg6hk7jzweyb3q2y7wkxreo2l6uu","attributes":[{"trait_type":"version","value":"0.1.0"}]}
```

Comment:
"The scaffold contract scaffolds a contract to be implemented by the developer." -> you need to update the metadata of this package in its yaml @jmoreira-valory